table/block_based/block_based_table_factory.cc

```diff
@@ -232,8 +232,6 @@ static std::unordered_map<std::string, OptionTypeInfo>
     block_based_table_type_info = {
 #ifndef ROCKSDB_LITE
         /* currently not supported
-          std::shared_ptr<Cache> block_cache = nullptr;
```

Comment:
Please keep this line, to remind us that block_cache is not dynamically configurable